### ISSUE TITLE
When the team gains upgrade-swords, the weapons in the enderchest will not gain sharpness

### ISSUE THREAD

**Issue by PixelEast** [andrei1058/BedWars1058#783]
When the team gains upgrade-swords, the weapons in the chest and enderchest will not gain sharpness

**To Reproduce**
1. Place any sword into the Enderchest
2. Purchase team upgrade upgrade-swords
3. Opening the EnderChest, the sword inside did not gain sharpness

**Expected behavior**
The weapons in Chest and EnderChest should be correctly sharpened

**System (please complete the following information):**
 - OS: Windows Server
 - Server software: Paper
 - Server Version: 1.12.2
 - Plugin Version: 23.3

**Comment by Lsy-291** [comment]
Perhaps you could try taking out the sword again?

**Comment by PixelEast** [comment]
> Perhaps you could try taking out the sword again?

not work
![GIF 2023-5-22 17-26-39](https://github.com/andrei1058/BedWars1058/assets/30312760/b2effbba-06c0-496a-9954-b7180210233f)

**Comment by PixelEast** [comment]
> Perhaps you could try taking out the sword again?

I tried to take it out, even drop it out and pick it up again, but it still won't have an enchanting effect
I think this is indeed a bug

**Comment by Lsy-291** [comment]
It seems to be true, perhaps you can try other server cores or replace the server version, or check your configuration files. By the way, are you a Chinese person

**Comment by Lsy-291** [comment]
Do you use QQ? Perhaps you can contact me through QQ to get some help. My QQ number is 2644898922

**Comment by andrei1058** [comment]
I think we missed this implementation.
Will be fixed someday!

**Comment by Lsy-291** [comment]
Oh, he doesn't seem to be related to the server. I used the original spigot 1.8.8 core test, but the issue still exists

**Comment by Lsy-291** [comment]
> I think we missed this implementation. Will be fixed someday!

Yes, this is not a bug. It's just that when you enchant a player's sword, you only traverse the player's inventory. I added enchantment to the sword in the enderchest in #797
